angular/src/app/book-table/components/book-table-dialog/book-table-dialog.component.spec.ts

```diff
@@ -8,15 +8,15 @@ import {
 import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 import { EffectsModule } from '@ngrx/effects';
 import { StoreModule } from '@ngrx/store';
-import { CoreModule } from 'app/core/core.module';
+import {CoreModule} from '../../../core/core.module';
```

Comment:
Hi @chdattat, I think that you are not using VS Code devonfw package or there is some config errors with Prettier since I see code that has not been linted nor prettified like this. 

In your own PoCs you can do whatever you want but My Thai Star should be better presented. The curly braces should have an space between for example. Please check this in the rest of files.

Reply:
Hi @sjimenez77, 

I am using VS Code from devonfw packages. Actually, it fixes all the lint issues automatically when we save the files due to this i neglect to run ng lint.  

So I am running ng lint explicitly but I can see many lint issues in most of the files. It is taking more time to fix the lint issues, process is going on. 

Let me know, do i commit these changes. 

![image](https://user-images.githubusercontent.com/29056094/94110076-a6158d80-fe5f-11ea-96f7-770fb14ed29b.png)

Reply:
Hi @sjimenez77,

After processing ng lint successfully, I am getting an error **"Property allowedCommonJsDependencies is not allowed."** in angular.json file. Might be configuration differences in vscode. I had compared both the files but there is no difference. 

I am unable to find out why "allowedCommonJsDependencies"  property is not allowed in angular.json after fixing lint issues.

So I reverted the changes and just fixed the space issues and tested. It is working as expected. And also I observed in Travis-ci build, there are no lint issues.

Please review it.

![image](https://user-images.githubusercontent.com/29056094/94146512-aaf33500-fe91-11ea-8a59-a5b720904152.png)


Thanks,
Dattu